### PR TITLE
platform-independent CMakeLists

### PR DESCRIPTION
**Pull request by r0ost3r**

### CODE REVIEW

**Comment by r0ost3r** [comment]
@Necktschnagge 
FYI
https://stackoverflow.com/questions/15639781/how-to-find-the-qt5-cmake-module-on-windows
https://lostdomain.org/2019/12/27/using-azure-devops-pipelines-with-qt/

**bitte
 azure pipeline anwerfen**

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+#rm -r build/
+#mkdir build/
+#cd build/
+
+#cmake ..
+#cmake --build .
+
+cd script/
+./uccpb-cmd-clean-create-project-build.sh
```
Comment (Necktschnagge) [inline build.sh:11]: I named the script the wrong way.
It must be `uccpb-unix-*`

Comment (Necktschnagge) [reply build.sh]: Halt warte, ich glaube du hast die geändert...

Comment (Necktschnagge) [reply build.sh]: Wolltest du die Shebang `#!/bin/bash' weg haben oder was war das Problem?

**Comment by Necktschnagge** [comment]
https://github.com/Necktschnagge/cmake-qt-prototype/pull/12

I removed your first item of steps. It is buggy.
Does it have any specific purpose?
I think it just does nothing...

**Comment by Necktschnagge** [comment]
This is obsolete since we merged #14.

**Pull request closed without merging**